uber/models/attendee.py

```diff
@@ -550,6 +553,17 @@ def _use_promo_code(self):
                 self.overridden_price = self.badge_cost
             else:
                 self.paid = c.NEED_NOT_PAY
+                
+    @presave_adjustment
+    def add_purchased_items(self):
```

Comment:
Ideally we'll beef this up later so it will also _update_ `purchased_items` as attendees make changes, but that was out of scope for this. I haven't decided yet if this should be a table rather than a column, anyway.